scripts/superdesk-search/services/TagService.js

```diff
@@ -180,15 +198,18 @@ export function TagService($location, desks, userList, metadata, search, ingestS
             }
 
             initParameters(tags.currentSearch);
+            initRemovedFacets(tags.currentSearch);
 
             _.forEach(tags.currentSearch, function(type, key) {
-                if (key !== 'q') {
+                if (key !== 'q' && !EXCLUDE_AGGREGATES[key]) {
```

Comment:
`!EXCLUDE_AGGREGATES.hasOwnProperty(key)` would be more correct here, to avoid unexpected behaviour.